ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -389,18 +404,47 @@ func (h *FaultHandler) StartNetworkPacketLoss() func(http.ResponseWriter, *http.
 		rwMu.Lock()
 		defer rwMu.Unlock()
 
-		// TODO: Check status of current fault injection
-		// TODO: Invoke the start fault injection functionality if not running
-
-		responseBody := types.NewNetworkFaultInjectionSuccessResponse("running")
-		logger.Info("Successfully started fault", logger.Fields{
+		var stringToBeLogged string
```

Comment:
non-blocking - I would set `Failed to start fault` to be the default value since it's used for most cases below.

Reply:
Addressing in the next revision